src/oc_erchef/habitat/config/sys.config

```diff
@@ -0,0 +1,345 @@
+                {file, "{{pkg.svc_var_path}}/logs/erchef.log"},
+                {level, info},
+                {size, 104857600},
+                {date, "$D0"},
+                {count, 10},
+                {formatter_config, [date, " ", time, " [", severity, "] ", message, "\n"]}
+            ]}
+        ]},
+
+        %% Whether to write a crash log, and where.
+        %% Commented/omitted/undefined means no crash logger.
+        {crash_log, "{{pkg.svc_var_path}}/logs/crash.log"},
+
+        %% Maximum size in bytes of events in the crash log - defaults to 65536
+        {crash_log_msg_size, 65536},
+
+        %% Maximum size of the crash log in bytes, before its rotated, set
+        %% to 0 to disable rotation - default is 0
+        {crash_log_size, 104857600},
+
+        %% What time to rotate the crash log - default is no time
+        %% rotation. See the lager README for a description of this format:
+        %% https://github.com/basho/lager/blob/master/README.org
+        {crash_log_date, "$D0"},
+
+        %% Number of rotated crash logs to keep, 0 means keep only the
+        %% current one - default is 0
+        {crash_log_count, 10},
+
+        %% Whether to redirect error_logger messages into lager - defaults to true
+        {error_logger_redirect, true},
+
+        %% Bump up the "high-water mark" (default 50), which is the
+        %% number of messages per second allowed to come from
+        %% error_logger.  This is the same as used by
+        %% opscode-chef-mover, FWIW.
+        {error_logger_hwm, 1000}
+    ]},
+
+                {chef_secrets, [
+                 {provider, chef_secrets_env}
+                ]},
+
+                {darklaunch, [
+               {config, "{{pkg.svc_data_path}}/dark_launch_features.json" },
+               {reload_time, 10000 }
+              ]},
+
+    {oc_chef_wm, [
+        {ip_mode, [ ipv4 ] },
+        {api_version, "12.0.0"},
+        {server_flavor, "cs"},
+
+        {default_orgname, undefined },
+
+        {ip, "0.0.0.0"},
+        {port, 8000},
+        {reqid_header_name, "X-Request-Id"},
+        {auth_skew, 900},
+        %% currently only used by the search endpoint to bound
+        %% how many nodes are deserialized at a time in
+        %% preparing a response.
+        {bulk_fetch_batch_size, 5},
+        {superusers, [<<"pivotal">>]},
+        %% metrics config
+        {root_metric_key, "chefAPI"},
+
+        {authz_timeout, 2000},
+        {authz_fanout, 20},
+
+        {reindex_batch_size, 10},
+
+        {enable_actions, false},
+        {ldap, []},
+        %% these are used for reporting on license status on the
+        %% license endpoint; it would have been nice to give these
+        %% their own logical section, but erlang requires these to
+        %% be part of a loaded application
+        {node_license, 999999},
+        {upgrade_url, <<"http://www.chef.io/contact/on-premises-simple">>},
+        {actions_host, "127.0.0.1"},
```

Comment:
I think we want actions permanently off int his setup so maybe we can remove all this stub config?